cmd/hatchet-migrate/migrate/migrations/20250829173445_v1_0_41.sql

```diff
@@ -0,0 +1,526 @@
+-- +goose no transaction
+-- +goose Up
+-- +goose StatementBegin
+BEGIN;
+CREATE TABLE IF NOT EXISTS v1_lookup_table_partitioned (
+    tenant_id UUID NOT NULL,
+    external_id UUID NOT NULL,
+    task_id BIGINT,
+    dag_id BIGINT,
+    inserted_at TIMESTAMPTZ NOT NULL,
+
+    PRIMARY KEY (external_id, inserted_at)
+) PARTITION BY RANGE (inserted_at);
+
+SELECT create_v1_weekly_range_partition('v1_lookup_table_partitioned', NOW()::DATE);
+SELECT create_v1_weekly_range_partition('v1_lookup_table_partitioned', (NOW() - INTERVAL '1 week')::DATE);
+
+DO $$
+DECLARE
+    startDateStr varchar;
+    endDateStr varchar;
+    targetTableName CONSTANT varchar := 'v1_lookup_table_partitioned';
+    newTableName varchar;
+BEGIN
+    SELECT '19700101' INTO startDateStr;
+    SELECT TO_CHAR(date_trunc('week', (NOW() - INTERVAL '8 days')::DATE), 'YYYYMMDD') INTO endDateStr;
+    SELECT LOWER(FORMAT('%s_%s', targetTableName, startDateStr)) INTO newTableName;
```

Comment:
Just to confirm, anything older than 8 days goes into the same partition?

Reply:
yeah this was my version of "from the beginning of time until last week" just to make sure we catch everything